versions.props

```diff
@@ -1,17 +1,6 @@
 ch.qos.logback:logback-classic = 1.1.7
 # catch-all jackson version
 com.fasterxml.jackson.*:jackson-* = 2.10.1
-# individual components are sometimes updated out of band to fix vulnerabilities
```

Comment:
For future reference, a star constraint like the above will handle these hotfix releases just fine as well.
As in, it will try to find the given version for all components, but will fall back to the highest constrained version (of a bottom lib like `jackson-databind`) if the requested version doesn't exist.

In fact jackson itself features in gradle's integration tests for this feature: 
https://github.com/gradle/gradle/blob/5d5c27d35e276d674da9a28a4af972d79589b78d/subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/alignment/AlignmentIntegrationTest.groovy#L241-L292